HackGreenvilleEventsAPITestSuite/EventsAPI.json

```
@@ -35,12 +35,21 @@
 							"});",
 							"",
 							"// Functional Testing Verification",
-							"//Event Name Exists",
-							"    pm.test(\"Event Name is not empty and exists\", function () {",
-							"        \tfor (let i=0; i <responseData.length; i++) {",
-							"        \t\tpm.expect(responseData[i].event_name).to.not.be.empty;",
+							"",
+							"// Event Name Exists",
+							"pm.test(\"Event Name is not empty and exists\", function () {",
+							"    for (let i=0; i <responseData.length; i++) {",
```

```suggestion
							"    \tfor (let i=0; i <responseData.length; i++) {",
```
This is where the import fails, likely because there is character escaping that is missing.  The previous lines show this in effect from where it was repaired earlier from a fresh file import that I put in the repository.

---

I exported the JSON from Postman and committed a new change on this PR/